docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -1,20 +1,29 @@
 # Prebuilt workspaces
 
-## Overview
+Prebuilt workspaces (Prebuilds) allow template administrators to improve the developer experience by reducing workspace
+creation time with an automatically maintained pool of ready-to-use workspaces for specific parameter presets.
 
-Prebuilt workspaces let you pre-provision and maintain a pool of ready-to-use workspaces.
-Instead of creating a new workspace when a developer requests one, if a workspace matches a preset defined in the
-template parameters, Coder assigns an existing instance, reducing setup time significantly.
+The template administrator configures a template to be available with prebuilt workspaces, and then when a developer creates
+a new workspace with matching parameters, Coder assigns them an existing prebuilt instance.
+Prebuilt workspaces significantly reduce wait times, especially for templates with complex provisioning or lengthy startup procedures.
+
+Prebuilt workspaces are:
+
+- Created and maintained automatically by Coder to match your specified preset configurations.
+- Claimed transparently when developers request matching workspaces.
+- Monitored and replaced automatically to maintain your desired pool size.
 
 ## Prerequisites
 
-- Premium license
-- Use `coder/coder` Terraform provider `>= 2.3.0-pre2` in your template (**TODO: update with latest version**)
-- Enable the `workspace-prebuilds` [experiment](../../../reference/cli/server.md#--experiments)
+- [**Premium license**](../../licensing/index.md)
+- [**Template administrator privileges**](../../users/groups-roles.md)
```

Comment:
```suggestion
```

I think it's redundant to mention this since all template changes require this role, and it muddies the waters since it's listed alongside licenses and feature flags which are out of the scope of this role.